### PR TITLE
Move documentation to use template

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          name: Install AWS CLI
+          command: |
+            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+            unzip awscliv2.zip
+            sudo ./aws/install
+      - run:
+          name: Publish Docs Site to S3
+          command: |
+            cd ./dist
+            aws s3 sync ./ s3://pluto.docs.fairwinds.com --delete
```
Comment (baderbuddy) [inline .circleci/config.yml:25]: Looks like you missed changing this to Insights

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+workflows:
+  version: 2
+  release:
+    jobs:
+      - publish_docs:
+          filters:
+            branches:
+              only: master
```
Comment (baderbuddy) [inline .circleci/config.yml:34]: This repo changed over to main

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 ```
 
-## Releasing
-```
-npm run build
-git add .
-git commit -a -m "rebuild the site"
-git push
-```
+Changes will be automatically pushed live when merged to master.
```
Comment (baderbuddy) [inline CONTRIBUTING.md:12]: main

**Review by baderbuddy — COMMENTED** [review]
Awesome! It looks like the GitHub Action is still there though so both Circle and GitHub will try to do the build.

**Comment by rbren** [comment]
Good catches!

**Pull request merged**